ui/app/templates/components/oidc/key-form.hbs

```diff
@@ -31,7 +31,7 @@
   </div>
   {{! RADIO CARD + SEARCH SELECT }}
   <div class="box is-sideless is-fullwidth is-marginless has-top-padding-xxl">
-    <h4 class="title is-4">Allowed applications</h4>
+    <h2 class="title is-4">Allowed applications</h2>
```

Comment:
Same here, if we change to the HDS text component, this will need to be updated.